src/tests/ftest/dfuse/container_attrs.py

```diff
@@ -0,0 +1,113 @@
+"""
+  (C) Copyright 2020-2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import re
+
+from apricot import TestWithServers
+from dfuse_utils import get_dfuse, start_dfuse
+
+
+class DfuseContainerAttrs(TestWithServers):
+    """Check if the dfuse attributes of a container are properly managed.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        # obtain separate logs
+        self.update_log_file_names()
+
+        # Start the servers and agents
+        super().setUp()
+
+    def check_attrs(self, dfuse, container_name):
+        """Check if the DFuse attributes of a container are loaded
+
+        Check in the log file of the dfuse instance if it contains the DFuse attributes of a given
+        container.  It also checks the value of the attributes found.
+
+        Args:
+            dfuse (Dfuse): DFuse instance to check
+            container_name (str): Name of the container
+        """
+        log_file = dfuse.get_log_file()
+        attrs = self.params.get("attrs", f"/run/{container_name}/*")
+        for attr in [attr.split(':') for attr in attrs.split(",")]:
+            match = None
+            attr_re = re.compile(r"^.+\ssetting\s+'" + attr[0] + r"'\s+is\s+(\d+)\s+seconds$")
+            for line in log_file:
+                match = attr_re.match(line)
+                if match:
+                    self.assertEqual(
+                        attr[1],
+                        match.group(1),
+                        f"Unexpected value for attribute {attr[0]}: "
+                        f"want={attr[1]}, got={match.group(1)}")
+                    break
+            self.assertIsNotNone(match, f"Setting of attribute {attr[0]} not found")
+
+    def test_dfuse_container_attrs(self):
+        """Jira ID: DAOS-14698.
+
+        Test Description:
+            Create a container with DFuse attributes
+            Mount a DFuse mount point
+            Check the output of the DFuse log
```

Comment:
The real thing we should be testing for here is:

- Create a container
- Start dfuse and access the container
- Create a second container with `daos container create --path ... --attrs=...`
- Verifying that dfuse is using the correct attributes for the second container.

Reply:
This is what is done, just not accessing the container in the second step.
However, it is indirectly accessed when we are creating a second container with create --path.

Reply:
Ah, I hadn't seen the second test.  The log check then checks for the unique values in the json in the dfuse logs then if I'm reading this correctly.

Reply:
In the json file we are asking to create the second container with the following attributes:
`  attrs: dfuse-attr-time:42,dfuse-dentry-time:404`
Then we are looking in the DFuse log file for the following lines:
```
10/21-09:48:29.41 wolf-66 DAOS[279335/279341/0] dfuse INFO src/client/dfuse/dfuse_core.c:707 dfuse_cont_get_cache(0x7f2b2c012ee0) setting 'dfuse-attr-time' is 42 seconds
10/21-09:48:29.41 wolf-66 DAOS[279335/279341/0] dfuse INFO src/client/dfuse/dfuse_core.c:707 dfuse_cont_get_cache(0x7f2b2c012ee0) setting 'dfuse-dentry-time' is 404 seconds
```